specs/introduction.md

```diff
@@ -0,0 +1,57 @@
+# Introduction
+
+Sphinx is a protocol that aims to make the smart contract deployment process more secure, transparent, and efficient. This document provides an overview of Sphinx to serve as context for the rest of the specification.
+
+## Goals
+
+* **Secure**: Deployments with Sphinx are tamper-proof. This is crucial because of the mission-critical nature of smart contract deployments, where subtle changes in a deployment can cause significant issues after a system has been deployed.
```

Comment:
It's unclear what tamper proof actually means in this context. I think we need to give more detail on what the tool actually does to make it clear what the value of this is exactly. I.e we should explain that the tool enables deployments to be approved off-chain and then executed by an arbitrary third party. In that context, being tamper-proof is very important.

I think it's reasonable to include this context either in the main readme or in this document. I would probably do a combination of both.

Reply:
Will also come back to this after updating the main README